README.md

```diff
@@ -797,6 +797,82 @@ In either case:
     end
     ```
 
+### Nested conditionals
+
+* <a name="no-nested-conditionals"></a>
+  Avoid the use of nested conditionals for flow of control. 
+  ([More on this][avoid-else-return-early].) <sup>[[link](#no-nested-conditionals)]</sup>
```

Comment:
No clear to me why a random blog post is an authoritative source. Our styleguide is authoritative by and in itself, no need to link to external sources. 

If you do feel this matter is controversial enough to require external validation, maybe link to Kent Beck's implementation pattern book, and the section on guard clauses.

See this sample chapter http://ptgmedia.pearsoncmg.com/images/9780321413093/samplechapter/BeckCh_0321413091.pdf